files/settings-default.cfg

```diff
@@ -108,6 +108,10 @@ distant interior fog start = 0
 
 distant interior fog end = 16384
 
+# When disabled, the visible distance is larger on screen edges. Enabling
+# the setting makes fog distance accurate all around the player.
```

Comment:
Maybe this could be worded a little more clearly. It should also be added to the REST documentation.